src/Compilers/CSharp/Portable/Parser/LanguageParser.cs

```diff
@@ -10055,6 +10073,32 @@ private bool ScanParenthesizedImplicitlyTypedLambda(Precedence precedence)
             return false;
         }
 
+        private bool IsParenVarCommaSyntax()
```

Comment:
Consider making this a local function, as it's only used in the one case. The compiler standard naming for local functions is camelCase.